src/browser/components/Form.jsx

```diff
@@ -110,13 +110,9 @@ export const TextInput = props => {
   return <StyledSettingTextInput {...rest}>{children}</StyledSettingTextInput>
 }
 
-export const CheckboxSelector = props => {
-  return props.checked ? (
-    <StyledCheckbox type="checkbox" {...props} />
-  ) : (
-    <StyledCheckbox type="checkbox" {...props} />
-  )
-}
+export const CheckboxSelector = props => (
+  <StyledCheckbox type="checkbox" {...props} />
+)
```

Comment:
Is there some way where these are not equal? Esp. since styled checkbox is a simple input field and doesn't have any side effects?